media-ranker/spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe BooksController, type: :controller do
+
+  before(:each) do
+    @book = Book.create(title: "Harry Potter and the Sorcer's Stone", author: "J.K.Rolling", description: "The story of the boy who lived")
+  end
+
+  describe "GET 'index'" do
+   it "is successful" do
+     get :index
+     expect(response.status).to eq 200
+   end
+ end
+
+  describe "POST 'create'" do
+   it "successfuly redirects" do
+     post :create
+     expect(response.status).to eq 302
```

Comment:
You can also `expect(response).to redirect_to(book_path(@book))` to be more explicit with redirect assertions.